android/src/main/java/cz/ackee/extensions/android/DrawableExtensions.kt

```diff
@@ -10,7 +11,47 @@ import androidx.annotation.ColorInt
 
 // Extensions for creating drawables and other drawable-related helpers
 
+/**
+ * Background Drawable. Do support dark mode
+ */
 fun backgroundDrawable(
```

Comment:
I have a few points about this approach.

First you need to pass Context to this method which we wanted to avoid. If necessary, I would make it an extension on `Context` or `ViewGroup` but we were able to avoid that. Can't you check it from `AppCompatDelegate`? I think that Michal is doing it like that on FlashSport.

Second, nobody reads docs for these functions. If you want to distinguish between these two methods do it in the name pls.

Reply:
`AppCompatDelegate` won't help here, the only way to check darkmode is through using `Context`
5da8a946ab170da3271e6c0d1f53597bd0586f73